.github/workflows/inspektor-gadget.yml

```diff
@@ -415,6 +423,19 @@ jobs:
       if: startsWith(github.ref, 'refs/tags/v')
       run: |
         make -C integration push
+    - name: Login to Azure Container Registry
+      if: startsWith(github.ref, 'refs/tags/v')
+      uses: docker/login-action@v1
+      with:
+        registry: ${{ secrets.AZURE_REGISTRY_CONTAINER_REPO }}
+        username: ${{ secrets.AZURE_REGISTRY_USERNAME }}
+        password: ${{ secrets.AZURE_REGISTRY_PASSWORD }}
+    - name: Push Integration Test Image to Azure Container Registry
+      if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
I think it'll fail because there is not a `$REGISTRYtest.azurecr.io/gadget` created -> https://github.com/kinvolk/inspektor-gadget/blob/main/integration/Makefile#L15. 

I'm not sure how we should handle, different options I see are:

- create a second `...test` registry 
- don't push this image ti ACR 
- push on the same registry but appending a suffix (like `-test`)

@alban do you have any opinion?

Reply:
> I think it'll fail because there is not a `$REGISTRYtest.azurecr.io/gadget` created
>
I had tested it and it worked because it is not `$REGISTRYtest.azurecr.io/gadget` but `$REGISTRY.azurecr.io/gadgettest`. It means that `gadgettest` is a different repository (which does not need to already exist) within the same registry `$REGISTRY.azurecr.io`, where we are pushing the gadget container image (which uses the repository `gadget`). It is the same for the Docker registry.

Reply:
Maybe what is creating confusion is that I didn't use a specific secret to store the Azure registry. I mean, currently, we use the following secrets:

- `CONTAINER_REGISTRY`: The URL of the registry which we use to login.
- `CONTAINER_REPO`: The URL of the repository which we use to push the gadget container images and the integration test image after appending `test`.

Instead, for the Azure registry, I used one single secret named `AZURE_REGISTRY_CONTAINER_REPO` to store the repository which I used for both, login and push.

Reply:
I have no idea why I was confused. Probably I was thinking that the commands in the readme were to create a container **repository** instead of a **registry**, so what both repositories have to be created by hand, but it turns out that the registry is created manually and repositories are created automatically when pushing.